src/main/resources/config.yml

```diff
@@ -0,0 +1,8 @@
+config-version: 1.0.0
```

Comment:
Is it wise to have a config-version when this is not yet supported, even if just for demo purpose? May cause confusion later on.